### DISCUSSION TITLE
Can't get SSL working

### DISCUSSION THREAD

**Discussion by maxiepax** [dannagle/PacketSender#379] [General]
cat /etc/issue: Ubuntu 24.04 LTS
uname -a: 6.8.0-39-generic #39-Ubuntu SMP PREEMPT_DYNAMIC Fri Jul  5 21:49:14 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
PacketSender version: Packet Sender Version 8.7.1 / SSL:OpenSSL 1.1.1f  31 Mar 2020 / Commit: 1a43150

Hi,
I've got two VMs, (192.168.0.3 and 192.168.0.4) where i'm trying to simulate SSL traffic.

**192.168.0.3 - (Server)**
sudo ./Packet_Sender-v8.7.1-x86_64.AppImage --listen -s -b 443 -B 192.168.0.3
Binding to custom IP 192.168.0.3
SSL Server started on 192.168.0.3:443
Use ctrl+c to exit.

From: , Port:0
Response Time:2024-08-07 15:28:49.648

**192.168.0.4 - (Client)**
./Packet_Sender-v8.7.1-x86_64.AppImage -saw 500 192.168.0.3 443 "GET / HTTP/1.0\r\n\r\n"
Error: Failed to connect to 192.168.0.3

--

If i run the example in documentation **packetsender -saw 500 www.google.com 443 "GET / HTTP/1.0\r\n\r\n"**

It works fine and dumps the result, so it seems it's the Server end that isn't working correctly.
I've made sure that ufw (firewall) is disabled etc.
To make sure port 443 isn't for some magical reason being used, i also re-ran using port 5000, which the same result.

Am i doing something wrong to get SSL server working?

**Comment by dannagle** [comment]
What you are doing looks correct. It seems you found a bug in the Linux version. I am seeing the same problem on my side. It is possible something was not packaged correctly in the AppImage.
